cicd/benchmark.txt

```diff
@@ -1,25 +1,25 @@
-DATA: 19434458 valid lines. MARKUP: 74510 items
+DATA: 19434458 valid lines. MARKUP: 74638 items
```

Comment:
Following above `rm` command, i think the DATA count should be decreased..
Could you please check this again?

Reply:
not for just "benchmark" 
``rm`` is executed only for "performance benchmark"